src/mapml/control/ScaleBar.js

```diff
@@ -0,0 +1,48 @@
+export var ScaleBar = L.Control.extend({
+    options: {
+      metric: true,
+      imperial: true,
+      maxWidth: 100,
+      updateWhenIdle: true,
+      position: 'bottomleft'
+    },
+
+    onAdd: function (map) {
+      let container = L.DomUtil.create('div','leaflet-control-scale');
+
+      let scaleControl = L.control.scale(this.options);
+      scaleControl.addTo(map);
+      container.appendChild(scaleControl.getContainer());
+      container.title = "Scale bar";
+      container.setAttribute('tabindex',0);
+
+      this._container = container; 
```

Comment:
Unless the leaflet `L.Control.getContainer()` returns this container, I think you should create an overriding `M.ScaleBar.getContainer()`, so you can work with the container for this control.